css-ruby-1/Overview.bs

```diff
@@ -647,23 +647,25 @@ Ruby Layout</h2>
 	its [=base level=] is laid out on the line,
 	aligned according to its 'vertical-align' property
 	exactly as if its [=ruby bases=] were a regular sequence of inline boxes.
-	Each [=ruby base container=] is sized and positioned
-	to contain exactly all of its [=ruby bases=]’ margin boxes.
+	The [=content box=] of each [=ruby base container=] is sized and positioned
+	to contain exactly all of its [=ruby bases=]’ [=margin boxes=].
 
 	[=Ruby annotations=] associated with the [=base level=]
 	are then positioned with respect to their [=ruby base boxes=]
 	according to the applicable 'ruby-position' values.
 	[=Ruby annotations=] within a level (within a single [=ruby container=])
 	are aligned to each other as if they were inline boxes
 	participating in the same inline formatting context.
-	Each [=ruby annotation container=] is sized and positioned
-	to contain exactly all of its [=ruby annotations=]’ margin boxes.
+	The [=content box=] of each [=ruby annotation container=] is sized and positioned
+	to contain exactly all of its [=ruby annotations=]’ [=margin boxes=].
 
-	[=Ruby annotation containers=] are stacked outward
-	over or under their corresponding [=ruby base container=],
+	The [=content boxes=] of [=ruby annotation containers=] are stacked outward
+	over or under the [=content box=] of their corresponding [=ruby base container=],
```

Comment:
For reference, see also TTML2's [`tts:rubyPosition`](https://www.w3.org/TR/ttml2/#style-attribute-rubyPosition) which specifies whether under or over should be used for each line in the ruby text. If there is a similar CSS control for this it would be useful to reference it here, since there are use cases where it is important to specify the position.